package-lock.json

```diff
@@ -0,0 +1,9330 @@
+{
```

Comment:
Shall we add it to `.gitignore` then?

Reply:
Yes, and/or better yet, please add a `.npmrc` file like the following one, so that the `package-lock.json` file is not created in the first place: https://github.com/balena-io-modules/resin-multibuild/blob/v4.7.3/.npmrc